Parser/Node.cs

```diff
@@ -48,6 +59,30 @@ public record NWriteStmt (bool NewLine, NExpr[] Exprs) : NStmt {
 public record NAssignStmt (Token Name, NExpr Expr) : NStmt {
    public override T Accept<T> (Visitor<T> visitor) => visitor.Visit (this);
 }
+
+public record NReadStmt (Token[] List) : NStmt {
+   public override T Accept<T> (Visitor<T> visitor) => visitor.Visit (this);
+}
+
+public record NCallStmt (Token Name, NExpr[] Expr) : NStmt {
+   public override T Accept<T> (Visitor<T> visitor) => visitor.Visit (this);
+}
+
+public record NIfStmt (NExpr Expr, NStmt[] Stmts) : NStmt {
+   public override T Accept<T> (Visitor<T> visitor) => visitor.Visit (this);
+}

```

Comment:
Change this to:
`public record NIfStmt (NExpr Condition, NStmt IfPart, NStmt? ElsePart)`